### PR TITLE
Update Test Certificates to Expire in Year 2051

### LINKED ISSUE #1276: fhir-server-test relies on certificates in p12 files expire in April 2021
fhir-server-test relies on certificates in p12 files expire in April 2021

### PR DESCRIPTION
**Pull request by prb112**
- Update CI/CD Audit resources (jks)
- Update Minio Resources (crt,key)
- Update fhir-server trust and key store (p12)
- Update fhir-client (p12)
- Create build/certificates with documentation on certificates and utility scripts

Closes #1276

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: build/certificates/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# IBM FHIR Server - Development Certificates 
+
+The certificates are used in development and CI/CD.
+These certificates are not configured for production use.
+
+The sections in this document walks through creation of the certificates in a temporary folder, and then subsequent update inline.
```
Comment (tbieste) [inline build/certificates/README.md:6]: Change "walks" to "walk".

File: build/certificates/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+The certificates are used in development and CI/CD.
+These certificates are not configured for production use.
+
+The sections in this document walks through creation of the certificates in a temporary folder, and then subsequent update inline.  
+
+The Audit Kafka jks are updated separately. 
+
+# Create CA and Certs and P12s (does not replace)
+
+This script should be run first, and subsequently each many section must update their cert/key.
```
Comment (tbieste) [inline build/certificates/README.md:12]: Not sure what "each many section" means.

File: build/certificates/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+2. Set `export WORKSPACE=$(pwd)`
+
+3. Run `bash ${WORKSPACE}/build/certificates/create-ca-and-certs.sh` to update the certs.
+
+At the end the tmp folder contains a copy of the certs and keys with a CA Issuer of `Issuer: C=US, O=IBM Corporation, OU=IBM Watson Health, OU=FHIR`.
+
+# Minio: Continuous Integration Certificates
+
+The following certifiates are used with minio.
```
Comment (tbieste) [inline build/certificates/README.md:24]: Misspelling "certifiates" in multiple places in this file.

**Review by lmsurpre — APPROVED** [review]
LGTM pending troy's feedback on the doc

**Comment by tbieste** [comment]
Made a few documentation comments, otherwise LGTM.

File: build/certificates/README.md (hunk 2 above)
Comment (prb112) [reply build/certificates/README.md:12]: fixed

File: build/certificates/README.md (hunk 1 above)
Comment (prb112) [reply build/certificates/README.md:6]: fixed

File: build/certificates/README.md (hunk 3 above)
Comment (prb112) [reply build/certificates/README.md:24]: fixed

**Pull request merged**